dspace-api/src/main/resources/org/dspace/storage/rdbms/sqlmigration/metadata/h2/V7.0_2020.10.31__CollectionCommunity_Metadata_Handle.sql

```diff
@@ -0,0 +1,99 @@
+--
+-- The contents of this file are subject to the license and copyright
+-- detailed in the LICENSE and NOTICE files at the root of the source
+-- tree and available online at
+--
+-- http://www.dspace.org/license/
+--
+
+-- ===============================================================
+-- WARNING WARNING WARNING WARNING WARNING WARNING WARNING WARNING
+--
+-- DO NOT MANUALLY RUN THIS DATABASE MIGRATION. IT WILL BE EXECUTED
+-- AUTOMATICALLY (IF NEEDED) BY "FLYWAY" WHEN YOU STARTUP DSPACE.
+-- http://flywaydb.org/
+-- ===============================================================
+
+-------------------------------------------------------------
+-- This will create COMMUNITY handle metadata
+-------------------------------------------------------------
+
+insert into metadatavalue (metadata_field_id, text_value, text_lang, place, authority, confidence, dspace_object_id) 
+  select distinct 
+  	T1.metadata_field_id as metadata_field_id, 
+  	concat(T2.handleprefix, h.handle) as text_value, 
```

Comment:
If you go with the approach I suggested above, I think this line (and the same line in the other INSERT statement) would just become:
```
concat("${handle.canonical.prefix}", h.handle) as text_value,
```

Obviously, you can also remove other references to the T2 table in this script (namely the `cross join` below)

Reply:
@tdonohue  I followed the mentioned steps, It works. I only had to change double quotes with single quote since both postgres and h2 were failing (double quotes was interpreted as a column name).